src/server/utils/tx-manager/queueMutex.js

```diff
@@ -5,6 +5,7 @@ export default class queueMutex {
     this.nonce = null
     this.mutex = new Mutex()
     this.lastFail = null
+    this.getTransactionCount = () => 0
```

Comment:
@AlexeyKosinski What for do you create this function here? Is it really needed?

Reply:
it is just for initial value. when we init adminWallet we update this function to real web3 function